### PR TITLE
added ukrainian translation

### PR DESCRIPTION
**Pull request by zhil**

### CODE REVIEW

File: src/locale/ua.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
+//! moment.js locale configuration
+//! locale : russian (ru)
```
Comment (digz6666) [inline src/locale/ua.js]: Should change to Ukrainian (ua)

**Comment by zhil** [comment]
@digz6666 done

File: src/locale/ua.js (hunk 1 above)
Comment (zhil) [inline src/locale/ua.js]: @digz6666 done

**Comment by mj1856** [comment]
Please fix build errors and add tests.  Thanks.

**Comment by ichernev** [comment]
@zhil we already have Ukranian.  The correct language code is `uk`.
@mj1856 always verify the language code first :)

**Comment by mj1856** [comment]
@ichernev - oops!  good catch! :smile:

**Comment by zhil** [comment]
sorry

**Comment by zhil** [comment]
@mj1856 @ichernev 

just wonder - why do you use **uk** as country code for Ukraine?
valid code seems to be **ua** - https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2

**Comment by mj1856** [comment]
[ISO-3166](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code of Ukraine is `UA`
[ISO-639](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language code of Ukrainian is 'uk'

Language Code != Country Code

If you wanted to express a fully qualified [IETF language tag](https://en.wikipedia.org/wiki/IETF_language_tag), it would be `uk-UA`

Ukranian happens to only be spoken in Ukraine, but consider other languages that are spoken in multiple countries, such as `en-US` vs `en-GB` and also `es-ES` vs `es-MX`

**Comment by zhil** [comment]
@mj1856 
:+1: 

thanks a lot for explanation

**Comment by mj1856** [comment]
No problem, and thanks for participating!  Perhaps there are other areas you could assist.  We have lots of open issues. :smile:

**Comment by zhil** [comment]
@mj1856 I am not very good in js, but I will check if I could help in some tickets :)

**Pull request closed without merging**